cmake/modules/AddSwift.cmake

```diff
@@ -1759,6 +1767,7 @@ function(add_swift_library name)
       SDK ${sdk}
       ARCHITECTURE ${arch}
       DEPENDS ${SWIFTLIB_DEPENDS}
+      GYB_DEPENDS ${SWIFTLIB_GYB_DEPENDS}
```

Comment:
The normal `DEPENDS` won't regenerate gyb files if they're changed. This is actually no longer necessary for this PR, so I can remove it.